internal/provider/onepassword_item_data_source.go

```diff
@@ -184,6 +194,56 @@ func (d *OnePasswordItemDataSource) Schema(ctx context.Context, req datasource.S
 								},
 							},
 						},
+						"file": schema.ListNestedBlock{
+							MarkdownDescription: sectionFilesDescription,
+							NestedObject: schema.NestedBlockObject{
+								Attributes: map[string]schema.Attribute{
+									"id": schema.StringAttribute{
+										MarkdownDescription: fileIDDescription,
+										Computed:            true,
+									},
+									"name": schema.StringAttribute{
+										MarkdownDescription: fileNameDescription,
+										Computed:            true,
+									},
+									"content": schema.StringAttribute{
+										MarkdownDescription: fileContentDescription,
+										Computed:            true,
+										Sensitive:           true,
+									},
+									"content_base64": schema.StringAttribute{
+										MarkdownDescription: fileContentBase64Description,
+										Computed:            true,
+										Sensitive:           true,
+									},
+								},
+							},
+						},
```

Comment:
`[suggestion]`

Here's one last thing that would be worth doing, and apologies for not thinking about it sooner.

What would be helpful for maintainability purposes it to make this schema into a variable and then pass it to both the `section` and in the root parts of the schema. You can see the similar pattern being done with the `recipe` block. 

This would enable us to easily update this schema in both places by making only one change. This also ensures that the two blocks in the schema are consistent.